lib/ansible/modules/cloud/amazon/ecs_attribute.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: ecs_attribute
+short_description: manage ecs attributes
+description:
+    - Create, update, delete or list ECS container instance attributes.
+version_added: "2.3"
+author: Andrej Svenke (@anryko)
+requirements: [ boto, botocore, boto3 ]
+options:
+    cluster:
+        description:
+            - The short name or full Amazon Resource Name (ARN) of the cluster that contains the resource to apply attributes. If you do not specify a cluster, the default cluster is assumed
```

Comment:
All `descriptions:` should be full sentences, and therefore end with a full stop.